### ISSUE TITLE
Jailbreak saigon 10.2.1

### ISSUE THREAD

**Issue by ArasHussen** [iabem97/saigon#147]
Hello 
I jailbreaked my phone but after delete som tweak lost my cydia icon 
How to back cydia icon disappeared ios 10.2.1 saigon

**Comment by dawidekziaja** [comment]
All you need to do is hold the jailbreak button for a few seconds and then tap it again to reinstall Cydia.

**Comment by ArasHussen** [comment]
Sorry my cydia icon its lost not jailbreak
My phone its jailbroken but cydia icon hidden 😔

**Comment by dawidekziaja** [comment]
Yes, but if you do what I said it won't jailbreak again, it will reinstall Cydia on your device, so it will no longer be "hidden"

**Comment by ArasHussen** [comment]
Do that but dont back it

**Comment by ArasHussen** [comment]
I do that but dont back it

**Comment by Puneet51199** [comment]
Same Problem.Didnt Get back cydia After Pressing and Holding Jailbreak Button. Saigon Says already jailbroken But no Cydia app. I am Running On Iphone 6s with ios 10.2.1

**Comment by CosmicGunners** [comment]
you can watch this video to fix that problem i did this on my iphone 5s on ios 10.2.1 it actually worked for me give it a try. 
https://www.youtube.com/watch?v=AECDX96E2k8&list=PL1EkaZOvGAuHWkSnfQHYfgNWXHNkUCYh7&index=1&t=310s
